.github/workflows/release.yml

```diff
@@ -57,4 +57,21 @@ jobs:
       - name: Upload Release Assets to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
-          password: ${{ secrets.PYPI_UPLOAD_TOKEN }}
\ No newline at end of file
+          password: ${{ secrets.PYPI_UPLOAD_TOKEN }}
+
+  trigger_docs_pr:
+    name: Trigger schematisation checks PR workflow on threedi-docs
+    needs: [publish]
+    runs-on: ubuntu-latest
+    steps:
+      - name: install cURL
+        run: sudo apt update && sudo apt-get install -y curl
```

Comment:
Heb je gekeken of dit nodig was? In ubuntu-latest is een heleboel al geïnstalleerd, curl zou ik ook verwachten.
En anders is de "apt update" denk ik sowieso niet nodig (en die duurt lang).

Reply:
je hebt gelijk, net even in de workflow log gekeken
```
curl is already the newest version (7.81.0-1ubuntu1.15).
0 upgraded, 0 newly installed, 0 to remove and 25 not upgraded.
```